client/test_commands/test_altbn128_mixer_base.py

```diff
@@ -0,0 +1,148 @@
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000011000",
+        2),
+    int(
+        "0100000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000100001",
+        2),
+]
+
+COMMITMENTS = [
+    int(
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000001",
+        2),
+    int(
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000010",
+        2),
+]
+
+HSIG = int(
+    "1010000000000000000000000000000000000000000000000000000000000000"
+    "0000000000000000000000000000000000000000000000000000000000000000"
+    "0000000000000000000000000000000000000000000000000000000000000000"
+    "0000000000000000000000000000000000000000000000000000000000101111",
+    2)
+
+HTAGS = [
+    int(
+        "1100000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000110010",
+        2),
+    int(
+        "1110000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000000000"
+        "0000000000000000000000000000000000000000000000000000000000111011",
+        2),
+]
+
+VPUB = (0x5555555555555500, 0x00eeeeeeeeeeeeee)
+
+# 255                                         128         64           0
+# |<empty>|<h_sig>|<nullifiers>|<msg_auth_tags>|<v_pub_in>)|<v_pub_out>|
+RESIDUAL_BITS = int(
+    "101"  # h_sig
+    "010"  # nf_1
+    "001"  # nf_0
+    "111"  # htag_1
+    "110"  # htag_0
+    "0101010101010101010101010101010101010101010101010101010100000000"  # vin
+    "0000000011101110111011101110111011101110111011101110111011101110",  # vout
+    2)
+
+PACKED_PRIMARY_INPUTS = \
+    [ROOT] + COMMITMENTS + NULLIFIERS + [HSIG] + HTAGS + [RESIDUAL_BITS]
+
+
+def test_assemble_nullifiers(mixer_instance: Any) -> None:
+    # Test retrieving nullifiers
+    print("--- testing ", "test_assemble_nullifiers")
```

Comment:
`test_assemble_nullifiers` -> `assemble_nullifier_test` (same for the functions below)

Reply:
Should this not be the name of the test? `assemble_nullifier_test` is the method name I happen to have used to wrap the internal methods.  Those solidity method names are more of an implementation detail (I was actually about to try reusing the same names - not sure if that works in solidity), whereas I guess the original `test_assemble_nullifiers` refers to the name of the test (the python function name).

Reply:
I'll switch  it to: `print("--- test_assemble_nullifiers")`, which is hopefully a bit clearer.

Reply:
(changed to clearer print statements)

Reply:
> I'll switch it to: print("--- test_assemble_nullifiers"), which is hopefully a bit clearer.

Thanks, that's a bit clearer (the `testing` prefix suggested that we ran a function named `test_assemble_nullifiers` which was not the case.).

> (changed to clearer print statements. Just FYI, I tried to keep the same names, but solidity does not seem to support overloading in this way, so I've kept the actual entry points as ...._test in the wrapper contract.)

Yes, let's keep the function names as they are If you changed the logs as per your comment above, that's fine. Thanks.